Tests/FuzzilliTests/AnalyzerTest.swift

```diff
@@ -30,8 +30,8 @@ class AnalyzerTests: XCTestCase {
             for instr in program.code {
                 contextAnalyzer.analyze(instr)
             }
-
-            XCTAssertEqual(contextAnalyzer.context, ProgramContext.script)
+            print(contextAnalyzer.context)
+            XCTAssertEqual(contextAnalyzer.context, .script)
```

Comment:
Can you extend this test case a bit or add a second one? Something like 
- Initial context is script context
- Start a function, ensure context is `.script, .function`
- Start a loop, ensure context is `.script, .function, .loop`
- Start another function inside the loop, ensure context is now `.script, .function` (.loop is gone)
- close the inner function, ensure context is again `.script, .function, .loop`
- maybe also start a class definition